clang/lib/Sema/SemaDecl.cpp

```diff
@@ -10465,7 +10466,7 @@ Sema::ActOnFunctionDeclarator(Scope *S, Declarator &D, DeclContext *DC,
                                                 Previous))
           NewFD->setInvalidDecl();
       }
-    } else if (isMemberSpecialization && isa<CXXMethodDecl>(NewFD)) {
+    } else if (isMemberSpecialization && !FunctionTemplate) {
```

Comment:
Same question here: are we still doing the checks for an explicit specialization of a member function template?

Reply:
Yes